configs/sites/tier1/atlantis/mirrors.yaml

```diff
@@ -1,15 +1,17 @@
 mirrors:
   local-source:
     fetch:
-      url: file:///gpfs/fs1/neptune/spack-stack/source-cache
+      # TODO UPDATE?
```

Comment:
@areinecke Should we update the location of the spack source cache from `/gpfs/.../` to `/neptune_diagnostics/...` ?

Reply:
It is in `/neptune_diagnostics/spack-stack/source-cache`, we could move it to `/gpfs/neptune`, but neptune_diagnostic is mounted on portal so it's easier to rsync it with the DSRC.

Reply:
Good point. I'll update it and make sure that there the old mirror on /gpfs is removed if it is there at all

Reply:
Done